Sources/MIDIKit/MIDIKit.swift

```diff
@@ -315,12 +315,19 @@ public class MIDIClient {
         case serialPortOwnerChanged
         case ioError(device: MIDIDevice, error: MIDIError)
     }
+    
     public private(set) var ref: MIDIClientRef = 0
     public let name: String
     public weak var delegate: MIDIClientDelegate?
+
+    // MIDI Network support
+    public internal(set) var networkSession: MIDINetworkSession?
+    internal let bonjourService = BonjourService()
```

Comment:
`MIDIClient` is a lightweight wrapper around `MIDIClientRef`.
I see no reason that `BonjourService` and `MIDINetworkSession` should be stored inside a `MIDIClient`.
I would prefer if they would be in a separate class.

Reply:
Could be MIDINetworkClient.